pkg/kubelet/volume/empty_dir/empty_dir.go

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
I don't actually find any guidance from Go people here.  They say "use short names" (paraphrasing) but do not address the case of "so your file name is logically two words".  Can you link me to a reference?

There's at least one other set of files that use underbars in pkg names, and that ALSO has my fingerprints...